test/stress/child.js

```diff
@@ -0,0 +1,87 @@
+const https = require('https')
+const path = require('path')
+const pify = require('pify')
+const fs = pify(require('fs'))
+const uuid = require('uuid')
+const _ = require('lodash/fp')
+const { PerformanceObserver, performance } = require('perf_hooks')
+
+const utils = require('./utils')
+const variables = require('./utils/variables')
+
+var certificate = {}
+var connectionInfo = {}
+
+async function getCert (machineIndex) {
+  const key = fs.readFile(path.resolve(__dirname, 'machines', `${machineIndex}`, 'client.key'))
+  const cert = fs.readFile(path.resolve(__dirname, 'machines', `${machineIndex}`, 'client.pem'))
+
+  return Promise.all([key, cert]).then(values => {
+    return { key: values[0], cert: values[1] }
+  }).catch(err => {
```

Comment:
what do you think of this?
```suggestion
  return Promise.all([key, cert]).then([key, cert]=> {
    return { key, cert }
  }).catch(err => {
```